src/UncontrolledButtonDropdown.svelte

```diff
@@ -5,13 +5,15 @@
   let className = '';
   export { className as class };
   export let disabled = false;
-  export let direction = 'down';
+  // TEMP
+  // export let direction = 'down';
   export let group = false;
   export let nav = false;
   export let active = false;
   export let addonType = false;
   export let size = '';
-  export let toggle = undefined;
+  // TEMP
+  // export let toggle = undefined;
```

Comment:
If these are unused, could we just remove them vs commenting out with the `// TEMP`?

Also could you combine these unused props PRs to one if possible?

Reply:
Deleted. Cannot clone right now. I'll update all others.